src/Codeception/Module/Yii2.php

```diff
@@ -196,7 +199,9 @@ public function _before(TestInterface $test)
      */
     private function loadFixtures($test)
     {
-        if (method_exists($test, self::TEST_FIXTURES_METHOD)) {
+        if (empty($this->loadedFixtures)
```

Comment:
it empties all the fixtures after unloading them at line 222 when `cleanup` is true. so this line is meaningless when `cleanup` equals true.

on the other case this only forbids to load the fixtures defined in the `_fixtures()` method twice. thats intended for when you have two tests on the same cest class

```php
public function _fixtures()
{
     // ...
}

public function a()
{
    $I->haveFixtures(...); // would work normally.     
     // save some data
}

public function b()
{
    // reloading the fixtures defined in _fixtures() means the data saved by
    // test `a` will be lost

    $I->haveFixtures(...); // would work normally.
}
```